FRAuthenticator/FRAuthenticator/Model/Notification/PushNotification.swift

```diff
@@ -117,7 +117,11 @@ public class PushNotification: NSObject, NSSecureCoding, Codable {
         self.ttl = ttlDouble
         self.mechanismUUID = mechanismUUID
         
-        self.timeAdded = Date()
+        if let intervalString = payload["i"] as? String, let interval = Int64(intervalString) {
+            self.timeAdded = Date(milliseconds: interval)
```

Comment:
This needs to use: Date(timeIntervalSince1970:)

Reply:
The code is already using the Date extension that parses the date as Date(timeIntervalSince1970:). No need to change.